README.md

```diff
@@ -328,3 +328,17 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## Configure fixtures
+
+There is a directory by the name ```fixtures``` in root project directory
+
+It contains initial data for our app in ```initial_data.json``` file.
+
+To load this data execute the following command:
+
+```
+python manage.py loaddata fixtures/initial_data.json
+```
+
+This will populate you database with the data inside ```initial_data.json```
```

Comment:
I think this should be added as part of the instructions and should be placed Immediately after the "local dev" guide, I also suggest "Database Population" as the header.